recipe-client-addon/test/browser/browser_about_studies.js

```diff
@@ -0,0 +1,147 @@
+"use strict";
+
+Cu.import("resource://shield-recipe-client/content/AboutPages.jsm", this);
+
+function withAboutStudies(testFunc) {
+  return async function inner(...args) {
+    return BrowserTestUtils.withNewTab(
+      {gBrowser, url: "about:studies"},
+      async function newTabInner(browser) {
+        return testFunc(...args, browser);
+      }
+    );
+  };
+}
+
+function getStudyRow(doc, studyName) {
+  return doc.querySelector(`.study[data-study-name="${studyName}"]`);
+}
+
+compose_task(
+  withAboutStudies,
+  async function testAboutStudiesWorks(browser) {
+    ok(browser.contentDocument.getElementById("app"), "App element was found");
+  }
+);
+
+compose_task(
+  withPrefEnv({
+    set: [["extensions.shield-recipe-client.shieldLearnMoreUrl", "http://test/%OS%/"]],
+  }),
+  withAboutStudies,
+  async function testLearnMore(browser) {
+    browser.contentDocument.getElementById("shield-studies-learn-more").click();
+    await BrowserTestUtils.waitForLocationChange(gBrowser);
+
+    const location = browser.contentWindow.location.href;
+    is(
+      location,
+      AboutPages.aboutStudies.getShieldLearnMoreHref(),
```

Comment:
Eh, I guess it doesn't matter too much either way. I just figured, right now we're essentially verifying that code that called `getShieldLearnMoreHref()` produces the same result as `getShieldLearnMoreHref()`... :-)